browser/js/kit/kit.controllers.js

```diff
@@ -1,8 +1,23 @@
-app.controller('KitsController', function ($scope, allKits, allKitImages) {
-	
+app.controller('KitsController', function ($scope, allKits, allKitImages, AuthService, $uibModal, $log) {
+	AuthService.getLoggedInUser().then(function(user){
+		$scope.user= user;
+	})
 	$scope.kits = allKits
 	$scope.images = allKitImages
 	$scope.title= 'All Kits'
+	$scope.open = function (size) {
```

Comment:
But if they were to make the ng-show true could they potentially open the model and create a kit?